src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 7 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain 'testexe -returncode 0 && testexe -returncode 1 &' as a job
```powershell
Expected @('0', '1'), but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 151
151:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain 'testexe -returncode 1 && testexe -returncode 0 &' as a job
```powershell
Expected '1', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 151
151:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```
Experimental Feature: && and || operators - Feature-Enabled.Runs the statement chain '$x = testexe -returncode 0 && Write-Output "mice" &' as a job
```powershell
Expected @('0', 'mice'), but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 151
151:         $resultJob | Wait-Job | Receive-Job | Should -Be $Output
```
Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```